CHANGELOG.rst

```diff
@@ -4,6 +4,29 @@ vmware.vmware Release Notes
 
 .. contents:: Topics
 
+v1.11.0
+=======
+
+Minor Changes
+-------------
+
+- _module_pyvmomi_base - Make sure to use the folder param when searching for VMs based on other common params in get_vms_using_params
+- added vm_resource_info module to collect cpu/memory facts about vms
+- clients/_pyvmomi - adds explicit init params instead of using dict
+- clients/_rest - adds explicit init params instead of using dict
+- esxi_hosts - Add inventory host filtering based on jinja statements
+- esxi_hosts inventory - include moid property in output always
+- folder - migrate vmware_guest_powerstate module from community to here
```

Comment:
Is this possibly due to C&P error in the changelog fragment? Looks like it should be `vm_powerstate` instead of `folder`. Otherwise, this doesn't make much sense to me.

The relevant PR seems to be #114.